common/source/docs/common-lua-scripts.rst

```diff
@@ -254,13 +286,58 @@ The GPS library provides access to information about the GPS's on the vehicle.
 
 - :code:`first_unconfigured_gps()` - Returns nil or the instance number of the first GPS that has not been fully configured. If all GPS's have been configured this returns 255 if all the GPS's have been configured.
 
-Notify
-~~~~~~
+GCS (gcs:)
+~~~~~~~~~~
+
+- :code:`send_text(severity, "text")` - Will send the text variable or constant string with message severity . Severity level is :
+
++---------------+-----------+
+|Severity Level | Type      |
++---------------+-----------+
+|0              | Emergency |
++---------------+-----------+
+|1              | Alert     |
++---------------+-----------+
+|2              | Critical  |
++---------------+-----------+
+|3              | Error     |
++---------------+-----------+
+|4              | Warning   |
++---------------+-----------+
+|5              | Notice    |
++---------------+-----------+
+|6              | Info      |
++---------------+-----------+
+
+
+- :code:`set_message_interval(serial channel, message type, rate)` - Sets the message types update rate on SERIAL (serial channel). For example, gcs:set_message_interval(0, uint32_t(30), math.floor(1000000 / 2.0)) sets SERIAL0 rate for ATTITUDE message stream (30) to 2.0 Hz .
```

Comment:
Way to complex an example. First note that the rate is actually the time between sending the message in microseconds. Then to keep the example simple it should just be an argument of `500000` the math.floor stuff is to fix a type problem, that you only have by doing division, and it's needlessly making this seem more complicated.

Reply:
taken directly from the examples....but I simplified it....